ledger/controllers/surveyor.js

```diff
@@ -91,8 +91,16 @@ v2.read =
 
     surveyor = await server(request, reply, runtime)
     if (!surveyor) return
-
-    reply(underscore.extend({ payload: surveyor.payload }, surveyor.publicInfo()))
+    const {
```

Comment:
I think we can remove all ledger side changes, there isn't automatic propagation both ways so ledger will always report the surveyor is unfrozen

Reply:
gotcha